tests/test_completion.py

```diff
@@ -341,25 +342,18 @@ def test_path_completion_doesnt_match_wildcards(cmd2_app, request):
     # Currently path completion doesn't accept wildcards, so will always return empty results
     assert cmd2_app.path_complete(text, line, begidx, endidx) == []
 
-def test_path_completion_invalid_syntax(cmd2_app):
-    # Test a missing separator between a ~ and path
-    text = '~Desktop'
-    line = 'shell fake {}'.format(text)
-    endidx = len(line)
-    begidx = endidx - len(text)
-
-    assert cmd2_app.path_complete(text, line, begidx, endidx) == []
+def test_path_completion_expand_user_dir(cmd2_app):
+    # Get the current user
+    user = getpass.getuser()
```

Comment:
For some reason this is causing a unit test failure on the AppVeyor Windows CI build/test systems.  All unit tests pass for this branch on my Windows 10 VM for both Python 3.6 and 2.7.  

But for some reason this is hosed on AppVeyor.   As far as I can tell, the problem is with AppVeyor and not with the code.  

If you can't find a good workaround quickly, I recommend just skipping this test on Windows.

Reply:
For some reason ``getpass.getuser()`` fails when running on AppVeyor. The user must not have any of these environment variables. Unfortunately the fallback imports the ``pwd`` module which doesn't exist on Windows.

```Python
for name in ('LOGNAME', 'USER', 'LNAME', 'USERNAME'):
            user = os.environ.get(name)
            if user:
                return user

# If this fails, the exception will "explain" why
import pwd
return pwd.getpwuid(os.getuid())[0]
```

I am trying a workaround.